### PR TITLE
Mercurial to Git updates:

### PR DESCRIPTION
**Pull request by aemerick**
- Changing .hgignore into .gitignore
- Removing mercurial requirements in SConstruct in favor of git,
  and changing commands to grab current changeset

### CODE REVIEW

**Comment by aemerick** [comment]
Thanks for the recommendation @jobordner . I generalized this now

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,19 @@
+Dan_test
```
Comment (brittonsmith) [inline .gitignore]: Is this a real thing?

Comment (aemerick) [reply .gitignore]: Ya... Not sure, it was in the original .hgignore though....

Comment (brittonsmith) [reply .gitignore]: I grepped around for anything that might make something like this and couldn't find anything. I think you get rid of it.

**Pull request merged**